.github/workflows/publish-canary.yml

```diff
@@ -75,6 +75,7 @@ jobs:
         uses: changesets/action@v1
         id: changesets
         with:
+          createGithubReleases: false
           publish: yarn changeset publish --tag canary
```

Comment:
oh good call @roninjin10 the canary snapshot one

Reply:
Yea not sure I think line 70 might need it I just approved to @trianglesphere figure it out

Reply:
Added to the full release. TY for the find.